apps/sample-app/README.md

```diff
@@ -1,11 +1,36 @@
 # Sample App
 
-Sample app leveraging teams js v2 for multi host experience
+The sample app leverages teams js v2 SDK for creating a multi host app experience. The sample app displays the usage of the following teams js v2 capabilties:
```

Comment:
Capabilities are the namespaces the APIs are in, so I think these would be described more accurately as "APIs"